mcs/class/referencesource/System/compmod/system/diagnostics/DiagnosticsConfiguration.cs

```diff
@@ -168,8 +168,10 @@ internal static class DiagnosticsConfiguration {
         }
         
         private static SystemDiagnosticsSection GetConfigSection() {
-            SystemDiagnosticsSection configSection = (SystemDiagnosticsSection) PrivilegedConfigurationManager.GetSection("system.diagnostics");
-            return configSection;
+            object o = PrivilegedConfigurationManager.GetSection("system.diagnostics");
```

Comment:
Maybe nit, but couldn't you just do
`return PrivilegedConfigurationManager.GetSection("system.diagnostics") as SystemDiagnosticsSection`